objmodel/chapter.md

```diff
@@ -1,86 +1,86 @@
-
+<!-- English spelling: "behaviour" line 46 -->
```

Comment:
I was confused by that comment. is that about British vs. American spelling? I'm usually trying to do American (see all the -ize), particularly since British requires "object-orientated" which is just weird :-).